src/views/Record.vue

```diff
@@ -5,115 +5,121 @@
         <div class="panel-heading">Loading...</div>
       </div>
       <div v-if="status.errored" class="panel panel-danger">
-        <div class="panel-heading">Something went wrong</div>
+        <div class="panel-heading">{{ status.error_title }}</div>
         <div class="panel-body">
-          <p>{{ status.error_message.message }}</p>
-          <pre>{{ status.error_message }}</pre>
+          <p>{{ status.error_message }}</p>
         </div>
       </div>
-
-      <div class="discover-full-record-basic-info">
-        <div class="record-image">
-          <img :alt="'cover for ' + result.title" />
-        </div>
-        <h2 class="record-title">
-          <span class="sr">Title: </span>{{ result.title }}
-        </h2>
-        <p>
-          <span v-if="result.content_type" class="record-type"
-            ><span class="sr">Type: </span>{{ result.content_type }}</span
-          >
-          <span v-if="result.publication_date" class="record-year"
-            >Published {{ result.publication_date }}</span
-          >
-        </p>
-        <p v-if="result.contributors" class="record-authors">
-          <span v-if="result.contributors.length > 1" class="sr"
-            >Authors:
-          </span>
-          <span v-else class="sr">Author: </span>
-          <span
-            class="record-author"
-            v-for="contributor in result.contributors"
-            v-bind:key="contributor.value"
-          >
-            <a data-type="Author" href="#">{{ contributor.value }}</a>
-          </span>
-        </p>
-      </div>
-      <div class="discovery-full-record-availability">
-        <h3 class="section-title sr">Availability</h3>
-      </div>
-      <div class="discovery-full-record-more-info">
-        <h3 class="section-title">More information</h3>
-        <ul class="list-moreinfo">
-          <li>
-            <span class="label">Document type: </span>{{ result.content_type }}
-          </li>
-
-          <li v-if="result.contributors">
-            <span v-if="result.contributors.length > 1" class="label"
+      <div v-else>
+        <div class="discover-full-record-basic-info">
+          <div class="record-image">
+            <img :alt="'cover for ' + result.title" />
```

Comment:
I think we should use the same logic in line 19 here, so the alt text displays "Unknown title" if `result.title` doesn't exist.

Reply:
I agree, will fix.

Reply:
@jazairi I've added a commit to address your feedback and think it ended up with a better solution for handling the missing title anyway at the data level rather than the display level.

Reply:
@JPrevost Good call; I like this solution better. :shipit: